app/helpers/test_passages_helper.rb

```diff
@@ -0,0 +1,12 @@
+module TestPassagesHelper
+
+  def result_message
+    if @test_passage.success?
+      "<font color=\"green\">#{@test_passage.success_percent}%</font><br>
```

Comment:
https://blog.bigbinary.com/2017/08/23/new-syntax-for-tag-helpers-in-rails-5-1.html